videoclip.lua

```diff
@@ -217,6 +204,24 @@ local function notify(message, level, duration)
     mp.osd_message(message, duration)
 end
 
+------------------------------------------------------------
+-- System-dependent variables
+
+local os_type
```

Comment:
Okay, turns out this is a problem because os_type was used at the top to determine the video folder for mac

Reply:
b2dad49 is the best I got, maybe moving the utility functions to a separate file could make it cleaner? Thing for another day.